.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python
+    steps:
+      - checkout
+      - setup_remote_docker
+
+      - run:
+          name: Create version.json
+          command: |
+            printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' \
+            "$CIRCLE_SHA1" \
+            "$CIRCLE_TAG" \
+            "$CIRCLE_PROJECT_USERNAME" \
+            "$CIRCLE_PROJECT_REPONAME" \
+            "$CIRCLE_BUILD_URL" > version.json
```

Comment:
Using `>` instead of `|`, relieves you from the tedious append of `\` to each line. But either way is fine in the end.

Doing `| tee version.json` is a small improvement for debugging builds so one can see in the circleci console what is going into version.json. But again either way.